src/server/page_server.cpp

```diff
@@ -99,6 +103,20 @@ page_server::receive_log_page_fetch (cubpacking::unpacker &upk)
     }
 }
 
+void
+page_server::receive_data_page_fetch (cubpacking::unpacker &upk)
+{
+  if (prm_get_bool_value (PRM_ID_ER_LOG_READ_DATA_PAGE))
+    {
+        VPID vpID;

```

Comment:
The name of local variable should be all lowercase.
```suggestion
        VPID vpid;
```